src/components/addbutton.js

```diff
@@ -4,9 +4,9 @@ class AddButton extends React.Component{
 
     render() {
         return (
-          <button style={styles.add} type="button" >add</button>
+          <button style={styles.add} type="button" onClick={this.props.clickHandler}>add</button>
```

Comment:
2. Here I attach the function that was passed down (in part 1) to the `onClick` event of the `button`